### PR TITLE
Added Direction for Tooltip

### PR DESCRIPTION
**Pull request by maliksenpai**
I liked this project and I want to contribute this project, if there is anything you want me to do you can tell me otherwise I will try new features or fixed.

## Status

**READY**

## Description

 Added direction param for tooltip direction. There are 2 different value for this param; "top" and "bottom". With this param tooltip position will change and arrow direction change too.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore
- [ ] 🧪 Tests

### CODE REVIEW

File: lib/src/widgets/nes_tooltip.dart [hunk 1]
```diff
@@ -36,6 +46,9 @@ class NesTooltip extends StatefulWidget {
   /// The placement of the arrow of the tooltip.
   final NesTooltipArrowPlacement arrowPlacement;
 
+  /// The direction fo the arrow of the tooltip
```
Comment (erickzanardo) [inline lib/src/widgets/nes_tooltip.dart:49]:
```suggestion
  /// The direction for the arrow of the tooltip
```

**Comment by erickzanardo** [comment]
Thanks for the contribution, I left a small typo fix comment.

Could you add a screenshot on how the widget with the tooltip on the bottom looks like?

**Comment by maliksenpai** [comment]
Web screenshot:
image

**Comment by erickzanardo** [comment]
Closing in favor of #105

**Pull request closed without merging**